docs/bold-bi-auto-deployment.md

```diff
@@ -0,0 +1,982 @@
+
+6. If you have a DNS to map with the application, then you can continue with the following steps, else skip to **Step 11**. 
+
+7. Uncomment the host value and replace your DNS hostname with `example.com` in deploy_aks.yaml file in line **1442**.
+
+      ![DNS](images/deploy/aks/ingress_http_yaml.png)
+
+8. If you have the SSL certificate for your DNS and need to configure the site with your SSL certificate, follow the below step or you can skip to **Step 11**.
+
+9.  Run the following command to create a TLS secret with your SSL certificate.
+
+```sh
+kubectl create secret tls bold-tls -n bold-services --key <key-path> --cert <certificate-path>
+```
+
+10. Now, uncomment the `tls` section and replace your DNS hostname with `example.com` in ingress spec and save the file.
+
+       ![ingress DNS](images/deploy/aks/ingress_yaml.png)
+
+11. Now, run the following command to get the ingress IP address.
+
+      ```sh
+      kubectl get svc -n ingress-nginx
+      ```
+      Repeat the above command till you get the IP address in EXTERNAL-IP tab as shown in the following image. 
+      ![Ingress Address](images/deploy/aks/ingress_address.png) 
+
+12. Note the EXTERNAL-IP address and map it with your DNS, if you have added the DNS in **deploy_aks.yaml** file. If you do not have the DNS and want to use the application, then you can use the EXTERNAL-IP address.
+
+13. Replace your DNS or EXTERNAL-IP address in `<application_base_url>` place.
+
+    Ex:  `http://example.com`, `https://example.com`, `http://<external_ip_address>`
+    
+    ![App_Base_Url](images/deploy/aks/app_base_url.png) 
+    
+14. Read the optional client library license agreement from the following link.
+
+    [Consent to deploy client libraries](../docs/consent-to-deploy-client-libraries.md)
+    
+15. By default all the client libraries will be installed for Bold BI in Kubernetes. Still you can still overwrite them by mentioning the required libraries as comma separated like below in the environment variable noted from the above link.
+
+    ![Optinal_Lib](images/deploy/aks/optional_lib.png) 
+
+16. Now, run the following commands to deploy Bold BI in your kubernetes cluster.
+
+    ```sh
+    kubectl apply -f deploy.yaml
+    ```
+
+17. Use the following command to get the pods status.
+
+     ```sh
+    kubectl get pods -n bold-services
+     ```
+    ![Pod status](images/deploy/pod_status.png) 
+
+18. Wait till you see the applications in running state. Then use your DNS or EXTERNAL-IP address you got from below command to access the application in the browser.
+    
+     ```sh
+    kubectl get ingress -n bold-services
+    ```
+    ![Ingress Address](images/deploy/aks/ingress.png)
+
+    ![Browser_veiw](images/deploy/Browser_veiw.png) 
+    
+19. If you facing any Deployment Error,Click Proceed to the application startup page link and Please refer the following link for more details on configuring the application startup manually.
```

Comment:
If you encounter a Deployment Error, Click **Proceed** to the application startup page link, and Please refer to the following link for more details on configuring the application startup manually.

Reply:
concern addressed